guides/specification.md

```diff
@@ -513,6 +516,32 @@ Considering the previous scenario what happens is that:
 >     expect_same: facts.installed_rpm_version
 > ```
 
+### failure_message
+
+An optional failure message can be declare for every expectation.
+
+In case of an `expect` one, the failure message can interpolate `facts` and `values` present in the evaluation scope to provide more meaningful insights:
+
+```yaml
+expectations:
+  - name: awesome_expectation
+    expect: values.awesome_constant_value == facts.awesome_fact
+    failure_message: The expectation did not match ${values.awesome_constant_value}
+```
+
+The outcome of the interpolation will be available into `ExpectationEvaluation` inside the API response.
```

Comment:
Not a big a fan of future term. I would rephrase to `... interpolation is available into`